src/esp/gfx/RenderCamera.cpp

```diff
@@ -152,6 +156,11 @@ uint32_t RenderCamera::draw(MagnumDrawableGroup& drawables, Flags flags) {
   }
 
   MagnumCamera::draw(drawableTransforms);
+
+  // reset
+  if (renderingForObjectPicking_) {
+    renderingForObjectPicking_ = false;
+  }
```

Comment:
why not just set renderingForObjectPicking_ to false always?  1 less check.

Reply:
Thanks. Because most of the time, it does not need to be reset.